up.py

```diff
@@ -24,76 +27,92 @@ def cmd_run(command):
     # Escape special chars
     command = shlex.quote(command)
     command = shlex.split(command)
-    #maybe try catch here...
-    process = subprocess.Popen(command, stdout=subprocess.PIPE,stderr=subprocess.PIPE,shell=True)
+    # maybe try catch here...
+    process = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True
+    )
     stdout, stderr = process.communicate()
     stdout = stdout.decode().strip()
     stderr = stderr.decode().strip()
     status = process.returncode
 
     return status, stdout, stderr
 
+
 def checkResults(results):
     if results[0] != 0:
-        print('ran into an error.  Please check the docker logs for details')
+        print("ran into an error.  Please check the docker logs for details")
         print(results[2])
         exit(1)
 
+
 def checkPort():
     if not args.port:
-        print('please specify the local port to run the container on using the -p/--port flag.')
+        print(
+            "please specify the local port to run the container on using the -p/--port flag."
+        )
         exit(1)
     else:
         return True
 
+
 def setup():
-    print('building container:')
-    results = cmd_run('docker build -t {} .'.format(imageName))
+    print("building container:")
+    results = cmd_run("docker build -t {} .".format(imageName))
     checkResults(results)
-    
-    print('Migrating Database:')
-    results = cmd_run('sudo docker run --rm -it --entrypoint "./manage.py" {} "migrate"'.format(imageName))
+
+    print("Migrating Database:")
+    results = cmd_run(
```

Comment:
another case of adding whitespce making it somewhat harder to read, in my opinion